eras/shelley/impl/src/Cardano/Ledger/Shelley/TxBody.hs

```diff
@@ -47,7 +47,7 @@ module Cardano.Ledger.Shelley.TxBody (
   TxOut,
   ShelleyTxOut (ShelleyTxOut, TxOutCompact),
   Url,
-  Wdrl (..),
```

Comment:
We also need to add the export of deprecated type synonym and pattern synonym `Wdrl`